tests/test_expressions.py

```diff
@@ -52,3 +52,41 @@ def test_validate(self):
 
         with self.assertRaises(ValueError):
             exp.Hint()
+
+    def test_transform_simple(self):
+        expression = parse('IF(a > 0, a, b)')[0]
```

Comment:
i wonder if we want to add a new api parse_one or something like that or if [0] is fine everywhere

Reply:
I was wondering the same thing. I think having `parse_one` might be a good idea. I personally like it more than [0].

Reply:
ok let's add that and refactor

Reply:
Do you mean add that in this PR? Or a subsequent one? I'm ok with either way

Reply:
Never mind, I've already refactored this one. Will push the change soon.